devutils/lib/bash/bazel.bash

```diff
@@ -39,7 +39,11 @@ Bazel::TestTargets () {
 
     while read target
     do
-        bazel test --ignore_unsupported_sandboxing $target || return 1
+        $made_tests || Log::Notice "testing associated targets (this might take a while)"
```

Comment:
`made_tests` isn't defined previously with a defined value and a `local` keyword